frontend_tests/node_tests/templates.js

```diff
@@ -221,16 +221,17 @@ function render(template_name, args) {
     html += "</table>";
 
     var buttons = $(html).find('.button');
+    var is_admin = page_params.is_admin;
+    if (is_admin) {
```

Comment:
You generally want to avoid having conditionals in unit tests.  You can modify `page_params.is_admin` to be true here, and test those cases, then modify it to be false, and test those cases.  Since the gist of this change is making sure the templates render for two different cases, we probably want the tests to exercise both paths.